modules/configuring-the-codeflare-operator.adoc

```diff
@@ -16,15 +16,15 @@ endif::[]
 
 
 ifdef::upstream[]
-* You have installed the required distributed workloads components as described in link:{odhdocshome}/installing-open-data-hub/#installing-distributed-workload-components_installv2[Installing the distributed workloads components].
+* You have installed the required distributed workloads components as described in link:{odhdocshome}/installing-open-data-hub/#installing-the-distributed-workload-components_installv2[Installing the distributed workloads components].
```

Comment:
This one redirects to https://opendatahub.io/docs/installing-open-data-hub/#installing-odh-v2_installv2, but I'm guessing it still needs to be published.

Reply:
The upstream doc is not published yet, if that's what you're asking?

Reply:
The module hasn't been moved to its new location yet upstream, so the link https://opendatahub.io/docs/installing-open-data-hub/#installing-the-distributed-workload-components_installv2 doesn't work yet :)